src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -289,4 +350,78 @@ protected function args_to_context( array $arguments, Context $context ) {
 
 		return $context_args;
 	}
+
+	/**
+	 * Handles gathering the data for admin fields.
+	 *
+	 * @since TBD
+	 *
+	 * @param int                 $field_name    The ID of the field.
+	 * @param array<string,mixed> $field       The field info.
+	 * @param Context             $context     The Admin View current context.
```

Comment:
:no_entry_sign: Doc comment for parameter $context does not match actual variable name $field

<sub>hash: d32abd1937b8697a6e375a3f6c65fc5b</sub>